src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -18,19 +20,22 @@ public class DeleteCommandParser implements Parser<DeleteCommand> {
      */
     public DeleteCommand parse(String args) throws ParseException {
         String trimmedArg = args.trim();
-        try {
-            if (!trimmedArg.startsWith("id/")) {
-                throw new ParseException(
-                        String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE));
-            }
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_MODULE);
+        argMultimap.verifyNoDuplicateStudentId(args);
```

Comment:
The verification of no duplicate student IDs in the preamble is a good addition. It ensures data integrity.